aci/resource_aci_fvsubnet.go

```diff
@@ -132,20 +132,18 @@ func getRemoteSubnet(client *client.Client, dn string) (*models.Subnet, error) {
 
 func setSubnetAttributes(fvSubnet *models.Subnet, d *schema.ResourceData) (*schema.ResourceData, error) {
 	dn := d.Id()
-	d.SetId(fvSubnet.DistinguishedName)
+	ip := d.Get("ip").(string)
+	d.Set("ip", ip)
```

Comment:
We should compare ip with fvSubnetMap["ip"] first to make sure it is equivalent then do the d.Set

Something like: https://www.techieindoor.com/go-how-to-compare-ip-addresses-in-go/

Reply:
ok